test/features/OrbitalFeature_tests.jl

```diff
@@ -0,0 +1,31 @@
+using Test
+using DataFrames
+using ..ChemistryFeaturization.Atoms: elements
+using ..ChemistryFeaturization.FeatureDescriptor: OrbitalFeatureDescriptor
+using SparseArrays
+
+@testset "Encode-Decode" begin
+    ofd = OrbitalFeatureDescriptor()
+
+    @testset "Encode - with `default_ofd_encode`" begin
+        silica = AtomGraph(Float32.([0 1 0; 1 0 1; 0 1 0]), ["O", "Si", "Ar"])
```

Comment:
minor nitpick – can we call this variable something else? Silica doesn't have argon in it ;)

Reply:
sorry, was testing different elements and ended up forgetting to change this. I'll replace Argon with Oxygen.